### PR TITLE
Run MP Config 3.0 TCK

### PR DESCRIPTION
**Pull request by tevans78**

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/io.openliberty.microprofile.config.3.0.internal_fat_tck/publish/tckRunner/tck/pom.xml [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!-- Needed for ZOS. Defines the version of this artifact that arquillian-liberty-managed should use -->
+            <!-- This doesn't exist for JEE 9 so leaving it out for now, will have to see what happens on ZOS -->
+            <!-- <dependency>
+                <groupId>org.jboss.spec.javax.servlet</groupId>
+                <artifactId>jboss-servlet-api_3.0_spec</artifactId>
+                <version>1.0.2.Final</version>
+            </dependency>-->
```
Comment (tevans78) [inline dev/io.openliberty.microprofile.config.3.0.internal_fat_tck/publish/tckRunner/tck/pom.xml:50]: @Emily-Jiang This bit is commented out for now but could be an issue on zos

**Review by Emily-Jiang — APPROVED** [review]
LGTM

**Comment by tevans78** [comment]
#rub-libby-bot

**Comment by tevans78** [comment]
#run-libby-bot

**Pull request merged**